src/main/js/components/overlay/ColumnEditorOverlay.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import i18n from 'i18next'
+
+class ColumnEditorOverlay extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      name: this.props.name,
+      description: this.props.description
+    }
+  }
+
+  modifyName = (evt) => {
```

Comment:
`modifyName` / `modifyDescription` looks completely the same. Does it make sense to create a private helper function that returns this handler with `name` / `description` key respectively?